### PR TITLE
Changing toPairs to return an array

### PR DESCRIPTION
**Pull request by dalefrancis88**
For merging into 0.13

This work is to start the journey to complete https://github.com/evilsoft/crocks/issues/336

i'm unsure if the type sig for arr is `[ String ]` or `Array String`

### CODE REVIEW

File: src/Pair/toPairs.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+// toPairs : Object -> Array (Pair String a)
 function toPairs(obj) {
   if(!isObject(obj)) {
     throw new TypeError('toPairs: Object required for argument')
   }
 
-  return Object.keys(obj).reduce(
-    (acc, key) => obj[key] !== undefined
-      ? acc.concat(List.of(Pair(key, obj[key])))
+  return Object.entries(obj).reduce(
```
Comment (dalefrancis88) [inline src/Pair/toPairs.js]: entries is supported by most, if not all, browsers and Node these days, but let me know if this should go back to keys

File: src/Pair/toPairs.js [hunk 2]
```diff
@@ -1,21 +1,20 @@
 /** @license ISC License (c) copyright 2017 original and current authors */
 /** @author Ian Hofmann-Hicks (evil) */
 
-const List = require('../core/List')
 const Pair = require('../core/Pair')
 const isObject = require('../core/isObject')
 
-// toPairs : Object -> List (Pair String a)
+// toPairs : Object -> Array (Pair String a)
```
Comment (dalefrancis88) [inline src/Pair/toPairs.js]: `[ (Pair String a) ]` or `Array (Pair String a)`?

File: src/Pair/toPairs.js (hunk 1 above)
Comment (evilsoft) [reply src/Pair/toPairs.js]: Thinking we should go back to keys. JIC

File: src/Pair/toPairs.js (hunk 2 above)
Comment (evilsoft) [reply src/Pair/toPairs.js]: I am good with how you have it, reads better IMO

Comment (dalefrancis88) [reply src/Pair/toPairs.js]: I agree

Comment (dalefrancis88) [reply src/Pair/toPairs.js]: I agree

**Comment by dalefrancis88** [comment]
The idea of the 0.13 is to get the work for #336 completed and any other breaking changes but still merging/closing the branches. I think that work may take a bit of time so breaking it up incase multiple people want to work on pieces.

**Pull request merged**